.github/renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
While I am aware of the need of tekton pipeline, may I ask why are we adding renovate here?
We already have depenadbot integrated as well.

Reply:
Ack, thanks for clarifying